homeassistant/components/wmspro/config_flow.py

```diff
@@ -84,6 +84,15 @@ async def async_step_user(
                 if not pong:
                     errors["base"] = "cannot_connect"
                 else:
+                    await hub.refresh()
+                    rooms = set(hub.rooms.keys())
+                    for entry in self.hass.config_entries.async_entries(DOMAIN):
+                        if (
+                            entry.runtime_data
```

Comment:
Maybe add a check for `entry.state is ConfigEntryState.LOADED`

Reply:
For some reason my test config entry is not loaded so I cannot get it to properly detect the duplicate if this is added.

Reply:
Got it working now with a `await hass.async_block_till_done()` between config entries in the tests.